wcfsetup/install/lang/de.xml

```diff
@@ -4809,6 +4809,11 @@ sich{/if} nicht bei uns registriert {if LANGUAGE_USE_INFORMAL_VARIANT}hast{else}
 		<item name="wcf.user.reauthentication.explanation"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du betrittst einen besonders geschützten Bereich. Aus Sicherheitsgründen ist es notwendig, dass du dich durch Eingabe deines Kennworts erneut authentifizierst.{else}Sie betreten einen besonders geschützten Bereich. Aus Sicherheitsgründen ist es notwendig, dass Sie sich durch Eingabe Ihres Kennworts erneut authentifizieren.{/if}]]></item>
 		<item name="wcf.user.reauthentication.loginAs"><![CDATA[Angemeldet als]]></item>
 		<item name="wcf.user.reauthentication.logoutAndChangeUser"><![CDATA[Nicht {$__wcf->user->username}? <a href="{link controller='FullLogout' application='wcf'}t={csrfToken type='url'}{/link}">Abmelden und Benutzer wechseln</a>.]]></item>
+		<item name="wcf.user.ignore.type"><![CDATA[Benutzer blockieren]]></item>
+		<item name="wcf.user.ignore.type.noIgnore"><![CDATA[Nicht Blockieren]]></item>
+		<item name="wcf.user.ignore.type.blockDirectContact"><![CDATA[Direkten Kontakt blockieren]]></item>
+		<item name="wcf.user.ignore.type.hideMessages"><![CDATA[Erstellte Inhalte ausblenden]]></item>
```

Comment:
This language item does not indicate that direct contact is also blocked with this option (same for English).

Reply:
Any suggestions? Radio buttons unfortunately do not support descriptions and I want to keep the phrases succinct. I'd say that it is pretty much obvious that this includes “direct contact” to be useful.

Maybe simple prefix “Zusätzlich”?

Reply:
I do not think that repeating the previous phrase is a bad thing.

Reply:
![image](https://user-images.githubusercontent.com/209270/115717185-36766b00-a37a-11eb-856c-146f7cec7895.png)
?

I find this hard to scan, because of the common prefix between both options.

Reply:
Yes, I prefer it this way because it is clear and unambiguous.